### PR TITLE
Update note about non-existing SQL_MoreResults

### PR DESCRIPTION
**Pull request by Impact123**
`SQL_MoreResults` does not exist. The note should refer to `SQL_FetchMoreResults` instead. This was found by Naleksuh and Addie: https://discord.com/channels/335290997317697536/335290997317697536/793685290509139988

### CODE REVIEW

**Review by Naleksuh — APPROVED** [review]
You stole this from me

**Comment by ErikMinekus** [comment]
Pretty sure it should be `SQL_MoreRows`. `SQL_FetchMoreResults` is for fetching multiple result sets from a single query.

File: plugins/include/dbi.inc [hunk 1]
```diff
@@ -162,7 +162,7 @@ methodmap DBResultSet < Handle
 	// Fetches a row from the current result set.  This must be 
 	// successfully called before any results are fetched.
 	//
-	// If this function fails, _MoreResults can be used to
+	// If this function fails, _MoreRows can be used to
```
Comment (asherkin) [inline plugins/include/dbi.inc:165]: I think the underscore here was accidentally left over when porting the documentation to the methodmap, and shouldn't be there.

```suggestion
	// If this function fails, MoreRows can be used to
```

**Review by asherkin — APPROVED** [review]
LGTM, but it'd be good to get one additional minor change before merging.

**Pull request merged**